64_FFT/app_resources/shader.comp.hlsl

```diff
@@ -0,0 +1,45 @@
+#include "common.hlsl"
+#include "nbl/builtin/hlsl/workgroup/fft.hlsl"
+
+[[vk::push_constant]] PushConstantData pushConstants;
+
+groupshared output_t sharedmem[4 * WorkgroupSize];
```

Comment:
why 4x ?

Reply:
So output_t ended up being the scalar (because of some host code shenaningans that I could change anyway). But the thing is I'm using a `WorkgroupSize` amount of threads to compute a `2 * WorkgroupSize`-sized FFT. So that's 2 complex numbers, each comprised of 2 scalars, per invocation in the workgroup

Reply:
you only need half this